components/admin/admin-dashboard.tsx

```
@@ -2,273 +1,297 @@
+								key={tab.key}
+								role="tab"
+								aria-selected={activeTab === (tab.key as any)}
+								aria-controls={`section-${tab.key}`}
+								onClick={() => setActiveTab(tab.key as any)}
+								className={`px-4 py-2 rounded-full text-sm font-medium border transition-colors ${
+									activeTab === (tab.key as any)
+										? 'bg-theme-primary text-white border-theme-primary'
+										: 'bg-white dark:bg-gray-900 text-gray-700 dark:text-gray-300 border-gray-200 dark:border-gray-700 hover:bg-gray-50 dark:hover:bg-gray-800'
+								}`}
+							>
+								{tab.label}
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)